integration-cli/docker_cli_build_test.go

```diff
@@ -1246,6 +1246,33 @@ func TestBuildCopyWildcardNoFind(t *testing.T) {
 	logDone("build - copy wild card no find")
 }
 
+func TestBuildCopyWildcardInName(t *testing.T) {
+	name := "testcopywildcardinname"
+	defer deleteImages(name)
+	ctx, err := fakeContext(`FROM busybox
+	COPY *.txt /tmp/
+	RUN [ "$(cat /tmp/\*.txt)" = 'hi there' ]
+	`, map[string]string{"*.txt": "hi there"})
+
+	if err != nil {
+		// Normally we would do t.Fatal(err) here but given that
```

Comment:
@crosbymichael this "if" is just checking the setup of the fake context, not actually running the build.
If this step fails then a TON of other tests will fail because there's something seriously wrong with our test environment that we can't even create a dir/tree/files.

If you want what I could do is split this - build the context first and the use context.Add() to add this one special file and then we know that if _that_ fails it must be due to _just_ the * in the name. 

Thoughts?